cognite/client/beta.py

```diff
@@ -0,0 +1,8 @@
+from cognite.client._api.relationships import RelationshipsAPI
+from cognite.client._cognite_client import CogniteClient as Client
+
+
+class CogniteClient(Client):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.relationships = RelationshipsAPI(self._config, api_version="beta", cognite_client=self)
```

Comment:
The problem with doing this is that `api_version` replaced the version on the URL (see `_get_base_url_with_base_path` in `_api_client.py`). 

A possible solution is to add an if on `_get_base_url_with_base_path` to check if the version is `beta` and builds the url with `v1`. You'll also need a if on `_do_request` to add a header with the `headers["version"] = "beta"`.